src/EnergyPlus/HVACVariableRefrigerantFlow.cc

```diff
@@ -13082,6 +13104,11 @@ void VRFTerminalUnitEquipment::CalcVRF_FluidTCtrl(EnergyPlusData &state,
         }
     }
     // calculate sensible load met using delta enthalpy
+    auto const &thisDXCoil = state.dataDXCoils->DXCoil(this->HeatCoilIndex);
+    // yujie: correct mismatching in the air mass flow rate
+    if (thisDXCoil.TotalHeatingEnergyRate > 0.0) {
+        AirMassFlow = state.dataDXCoils->DXCoil(this->HeatCoilIndex).InletAirMassFlowRate;
+    }
```

Comment:
Why is there a mismatch in AirMassFlow? AirMassFlow is used in the LoadMet calculation on line 13112. That calculation should include any supply side AT Mixer flow if used on the supply side of this TU which would be different than the AirMassFlow used by the coil. See line 13082 above.

Reply:
Replace lines 13107 - 13111 with:

    AirMassFlow = state.dataLoopNodes->Node(VRFTUOutletNodeNum).MassFlowRate;

This is the code flow:

    SetAverageAirFlow(state, VRFTUNum, PartLoadRatio, OnOffAirFlowRatio); <-- sets TU inlet node mass flow
    call fan and coils in correct order
    call ATMixer

So now here each child component has passed air mass flow from it;s inlet to outlet all the way through the TU. And the outlet node of the TU should be the mass flow rate used to calculate LoadMet.

Reply:
I moved the AirMassFlow updates to after `SimDXCoil` where the coil gets its updated air flow rate value.